extensions/amp-date-picker/0.1/amp-date-picker.js

```diff
@@ -803,9 +807,7 @@ export class AmpDatePicker extends AMP.BaseElement {
         this.updateDateFieldFocus_(this.dateField_);
       }
       this.transitionTo_(DatePickerState.OVERLAY_OPEN_INPUT);
-    } else if (this.element.contains(target)) {
```

Comment:
Yeah, I intended for it to keep the overlay open, but it doesn't need to transition to the `OVERLAY_OPEN_PICKER` state; it can just do nothing for that case instead.